src/app/projects/add-edit-project/form-tab-2018/form-tab-2018-component.spec.ts

```diff
@@ -1,6 +1,6 @@
-import { async, ComponentFixture, TestBed } from '@angular/core/testing';
-import { NO_ERRORS_SCHEMA } from '@angular/core';
-import { FormTab2018Component } from './form-tab-2018.component';
+// import { async, ComponentFixture, TestBed } from '@angular/core/testing';
```

Comment:
Can these be dropped?

Reply:
Yes. I would like to replace the dud test with a real one at some point, but that's probably not the point of the upgrade exercise.